libraries/standard/mqtt/include/mqtt.h

```diff
@@ -116,6 +116,17 @@ struct MQTTPubAckInfo
     MQTTPublishState_t publishState; /**< @brief The current state of the publish process. */
 };
 
+/**
+ * @brief The status codes in the SUBACK response to a subscription request.
+ */
+typedef enum MQTTSubAckStatus
```

Comment:
This results in two 2.3 violations since `MQTTSubAckStatus` and `MQTTSubAckStatus_t` are both defined but never used. Might need to add comments to suppress that.

Reply:
Will address that in a separate PR. Thanks for pointing out.